### PR TITLE
Create a UI component for a Calculator 🚀

### PR DESCRIPTION
**Pull request by hsztan**
I created a calculator 🧮 component that renders a graphical user interface 🖥 for a calculator that will later have functionality. 🤓

### CODE REVIEW

File: src/components/calculator/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import './calculator.scss';
+
+const Calculator = () => (
```
Comment (FrederickMih) [inline src/components/calculator/Calculator.jsx]: - [x] To fulfill one of the requirements of this project, let's make this component class-based.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @hsztan, 

You have done a nice job so far in this project. Keep up! :rocket: 
There are some issues that you still need to work on to go to the next project but you are almost there!
Screenshot from 2022-04-12 20-44-47



# To Highlights

-  Git/GitHub workflow has been followed :heavy_check_mark: 
-  Linters are correctly set up :heavy_check_mark: 
-  Pull Request has a title and summary :heavy_check_mark: 


## Required Changes ♻️

- [x] Hi @hsztan, please don't forget that it's a requirement to get rid of all the boilerplate that got created by CRA(texts, images, and styles). These are files are not necessary for your project. These files and images are found in the `src` directory and the `public` folder.
- [x] Please make the `App.js` component class-based. This way, you move a step forward towards project approval.
- [x] Again, the `App.js`(main component) should have imported the `Calculator.js` component to fulfill the requirements of this project. (**NB:** I'm leaving this comment here because you didn't make any changes in this file). Please fix this.

_Check the comments under the review._

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator/Calculator.jsx (hunk 1 above)
Comment (hsztan) [reply src/components/calculator/Calculator.jsx]: Hi @FrederickMih, ESLint doesn't let me change this functional component to a class component. What should I do?

Screen Shot 2022-04-19 at 2 07 56 PM

Comment (hsztan) [reply src/components/calculator/Calculator.jsx]: @FrederickMih, by the way. I also get the same linter issue when trying to convert App.js to a class-based module. I will have to convert them to functional components.

Screen Shot 2022-04-19 at 2 28 00 PM

Comment (FrederickMih) [reply src/components/calculator/Calculator.jsx]: Hi @hsztan. Please let's try writing the component this way.

```
class Calculator extends React.PureComponent {
  render() {
    return <div>{this.props.Calculator }</div>;
  }
}
```

Thanks :pray:

Comment (hsztan) [reply src/components/calculator/Calculator.jsx]: @FrederickMih, was able to create the class-based components by extending the class from `React.PureComponent` 💪

Comment (FrederickMih) [reply src/components/calculator/Calculator.jsx]: @hsztan, that's great :+1: 

Wish you all the best :pray:

**Review by FrederickMih — APPROVED** [review]
# Status: Approved :tada: :tada: :champagne: 
133696300-b506e476-dc1a-443e-935a-c6f434e788b6

Hi @hsztan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**